src/day03_BinarySearch_Array/ArrayBinarySearch.java

```diff
@@ -0,0 +1,59 @@
+package day03_BinarySearch_Array;
+
+public class ArrayBinarySearch {
+    public static void main(String[] args) {
+//        int[] numbers = intergerify(args);
+//        binarySearchBottenbruch()
+    }
+
+//    public static int[] intergerify(String[] input){
+//        int[] integers = new int[input.length];
+//        for(int i = 0; i < input.length; i++){
+//           integers[i] = Integer.parseInt(input[i]);
+//        }
+//        return integers;
+//    }
+
+    public static int binarySearchBottenbruch(int[] array, int target){
+        int L = 0;
```

Comment:
I'm pretty sure Steve covered single-letter variables in lecture, but I may be misremembering...  In any case, single-letter variables are not the best idea; they can be hard for other developers to read, they are not very informative, and they are very hard (almost impossible) to find in a text search.